jcl/src/java.base/share/classes/java/lang/J9VMInternals.java

```diff
@@ -171,6 +171,12 @@ private static void completeInitialization() {
 			Runtime.getRuntime().addShutdownHook(new Thread(runnable, "CommonCleanerShutdown", true, false, false, null)); //$NON-NLS-1$
 		}
 		/*[ENDIF] JAVA_SPEC_VERSION >= 9 */
+/*[IF CRAC_SUPPORT]*/
+		if (openj9.internal.criu.InternalCRIUSupport.isCRaCSupportEnabled()) {
+			// export java.base/jdk.crac unconditionally
+			J9VMInternals.class.getModule().implAddExports("jdk.crac"); //$NON-NLS-1$
```

Comment:
why not Module::addExports ?

Reply:
https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/Module.html#addExports(java.lang.String,java.lang.Module)
public [Module](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/Module.html) addExports​([String](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/String.html) pn, [Module](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/Module.html) other)

To export a package unconditionally, the `other` module must be a special [EVERYONE_MODULE](https://github.com/ibmruntimes/openj9-openjdk-jdk11/blob/69eac55903b27058534411d28ab712ea00698bd7/src/java.base/share/classes/java/lang/Module.java#L254) which is a private field within `Module` requiring reflection.
Current invocation avoids the reflection access.